core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpSamDAO.scala

```diff
@@ -118,7 +118,7 @@ class HttpSamDAO(baseSamServiceURL: String, serviceAccountCreds: Credential)(imp
   }
 
   override def getUserStatus(userInfo: UserInfo): Future[Option[RawlsUser]] = {
-    val url = samServiceURL + "/register/user/v2/info"
```

Comment:
This endpoint doesn't seem to exist in the Sam swagger nor in the Sam code itself